pkgs/applications/graphics/freecad/linkbranch.nix

```diff
@@ -0,0 +1,8 @@
+import ./generic.nix {
+  pname = "freecad-linkbranch";
+  version = "2023.01.31";
```

Comment:
There's newer versions available

Reply:
I figured I should go for the latest release version. Think I should go for the newer versions instead?

Reply:
Oh yeah, the newer releases are pre-releases
This version is good then